### DISCUSSION TITLE
Schema validation issues continue

### DISCUSSION THREAD

**Discussion by garysolebc** [CMSgov/price-transparency-guide#251] [Q&A] [answered]
Schema validation issues.
Although the schema was recently updated, it is still an invalid schema.  The reason for this, is that the $ref in the service code section, refers to a URL that is not JSON, but rather an HTML.
 "items": {
            "$ref": "https://www.cms.gov/Medicare/Coding/place-of-service-codes/Place_of_Service_Code_Set"
          },

Since this list is relatively finite, does it make sense to include these values in the schema itself, similar to the billing code type enum.  Have the actual descriptions in documentation, not in schema.
Alternatively, change the attribute name from "$ref" to just "ref"

**Comment by shaselton-usds** [comment]
@garysolebc Yup, definitely a placeholder -- you can see this starting to be addressed in the "reference-files" branch: https://github.com/CMSgov/price-transparency-guide/blob/reference-files/schemas/in-network-rates/in-network-rates.json#L190

You're right that there isn't much change with that list -- moving the list externally (such as into a common .json file within the project) might not make much sense outside of trying to keep the base schema cleaner.

**Comment by shaselton-usds** [comment, marked as answer]
The current schema should validate correctly against [JSON draft 07](http://json-schema.org/draft-07/schema), please let me know otherwise.
